ci/qemu/mxnet_requirements.txt

```diff
@@ -0,0 +1,7 @@
+urllib3<1.23,>=1.21.1
```

Comment:
Are these QEMU specific requirements? Maybe we could just use the general requirements file as otherwise we might risk updating this one in case something changes

Reply:
Where are the original MXNet requirements?